zerver/migrations/0082_index_starred_user_messages.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    atomic = False
+
+    dependencies = [
+        ('zerver', '0081_make_emoji_lowercase'),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            ' CREATE INDEX CONCURRENTLY'
+            ' zerver_usermessage_starred_message_id'
+            ' ON zerver_usermessage (user_profile_id, message_id)'
+            ' WHERE (flags & 2) != 0;',
+            reverse_sql='DROP INDEX zerver_usermessage_starred_message_id;'
```

Comment:
I think we might want to encourage folks to run the SQL before upgrading in the release notes (because otherwise they will do the migrations with downtime).  So we should support that, in the form of having our SQL do nothing in the case that the index already exists.